.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,37 @@
+// For format details, see https://aka.ms/vscode-remote/devcontainer.json or this file's README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.128.0/containers/hugo
+{
+	"name": "Hugo",
+	"build": {
+		"dockerfile": "Dockerfile",
+	},
+
+	"settings": { 
+		"git.alwaysSignOff": true,
```

Comment:
This fulfils the Dapr repo requirement to always provide the `-s` parameter with git commit when using git from VSCode.